src/PluginLoader.php

```diff
@@ -15,48 +15,50 @@
 use Contao\ManagerPlugin\Bundle\BundlePluginInterface;
 use Contao\ManagerPlugin\Config\ConfigPluginInterface;
 use Contao\ManagerPlugin\Config\ExtensionPluginInterface;
-use Contao\ManagerPlugin\Dependency\DependencyResolverTrait;
-use Contao\ManagerPlugin\Dependency\DependentPluginInterface;
-use Contao\ManagerPlugin\Dependency\UnresolvableDependenciesException;
 use Contao\ManagerPlugin\Routing\RoutingPluginInterface;
 
+/**
+ * This is a stub class which will be replaced during `composer install` or
+ * `composer update` unless Composer is run with the `--no-scripts` flag.
+ */
 class PluginLoader
 {
-    use DependencyResolverTrait;
-
     public const BUNDLE_PLUGINS = BundlePluginInterface::class;
     public const CONFIG_PLUGINS = ConfigPluginInterface::class;
     public const EXTENSION_PLUGINS = ExtensionPluginInterface::class;
     public const ROUTING_PLUGINS = RoutingPluginInterface::class;
 
     /**
-     * @var string
+     * @var array
      */
-    private $installedJson;
+    private $plugins;
 
     /**
      * @var array
      */
-    private $plugins;
+    private $disabled = [];
 
     /**
-     * @param string $installedJson
+     * @param string|null $installedJson
+     * @param array       $plugins
      */
-    public function __construct($installedJson)
+    public function __construct($installedJson = null, array $plugins = null)
```

Comment:
This needs to be `string $installedJson` (or maybe `?string $installedJson`).

Reply:
Fixed in bfc084025e12240524c7841e068d5128cdf4081b